tock/tock/templates/projects/project_detail.html

```diff
@@ -1,99 +1,100 @@
 {% extends "base.html" %}
 {% block navigation %}
-  {% include "_navigation.html" %}
+{% include "_navigation.html" %}
 {% endblock %}
 
 {% load project_tags %}
 {% load staticfiles %}
+{% load analytics %}
 
 {% block content %}
+<script src="{% static 'js/plotly-1.54.7.min.js' %}"></script>
+<script src="{% static 'js/components/save_csv.js' %}"></script>
 
 <h1><a href="{% url 'ProjectListView' %}">Tock Projects</a> / {{ object.name }}</h1>
 
-<ul>
-  <li>
-    <b> Project Lead: </b>
-{% if project.project_lead %}
-    <a href="mailto:{{ object.project_lead.email }}">{{ object.project_lead.first_name }} {{ object.project_lead.last_name }} </a>
-{% else %}
-    No project lead available
-{% endif %}
-  </li>
-  <li>
-    <b> Client: </b>
-    {{ object.accounting_code.code }}
-  <li>
-    <b> Billable Status: </b>
-    {% if object.accounting_code.billable %}
-      Billable
-    {% else %}
-      Non-Billable
-    {% endif %}
-  </li>
-  <li>
-    <b> Agreement URL: </b>
-{% if project.agreement_URL %}
-  <a href="{{ object.agreement_URL }}"> Google Drive folder </a>
-{% else %}
-  No agreement URL available
-{% endif %}
-  </li>
-  <li> <b> Alerts: </b>
-{% if object.alerts.exists %}
-    {% for alert in object.alerts.all %}
-        {% if alert.destination_url %}
-          <a href="{{ alert.destination_url }}" target="_blank"><span class="{{ alert.full_style }}">{{ alert.full_alert_text }}</span></a>
-        {% else %}
+<div class="grid-container">
+  <div class="grid-row">
+    <div class="tablet:grid-col">
+      <ul>
```

Comment:
Semantically I wonder if this is a worthy use of a `<dl>`? Instead of styling field names manually with `<b>`, could use `<dt>` and `<dd>`. But maybe it's not worth the effort of styling that infrequently used element?

Reply:
Likely and I'm definitely guilty here of not reworking the vast majority of the original html from `project_detail.html` and only made the changes necessary to get the charts rendering well-enough.

I'll file another issue to look at cleaning this up a bit, thanks @neilmb!